### PR TITLE
Add Postage Argument on Mint Command

### PR DESCRIPTION
**Pull request by ynohtna92**
Implements the postage argument for the wallet mint subcommand.

### CODE REVIEW

File: src/subcommand/wallet/mint.rs [hunk 1]
```diff
@@ -6,6 +6,11 @@ pub(crate) struct Mint {
   fee_rate: FeeRate,
   #[clap(long, help = "Mint <RUNE>. May contain `.` or `•`as spacers.")]
   rune: SpacedRune,
+  #[clap(
+    long,
+    help = "Amount of postage to include in the inscription. Default `10000sat`."
```
Comment (raphjaph) [inline src/subcommand/wallet/mint.rs:11]: This postage is for the mint of a rune, so no inscription inside. 

```suggestion
    help = "Amount of postage to include in the minted rune output. Default `10000sat`."
```

File: tests/wallet/mint.rs [hunk 2]
```diff
@@ -249,3 +253,71 @@ fn minting_rune_and_then_sending_works() {
...
+  );
+
+  core.mine_blocks(1);
+
+  let balance = CommandBuilder::new("--chain regtest --index-runes wallet balance")
+    .core(&core)
+    .ord(&ord)
+    .run_and_deserialize_output::<ord::subcommand::wallet::balance::Output>();
+
+  assert_eq!(balance.runic.unwrap(), 1000_u64);
```
Comment (raphjaph) [inline tests/wallet/mint.rs:322]: I don't think you need the `_u64`

File: tests/wallet/mint.rs [hunk 3]
```diff
@@ -249,3 +253,71 @@ fn minting_rune_and_then_sending_works() {
...
+      inscriptions: vec![batch::Entry {
+        file: "inscription.jpeg".into(),
+        ..default()
+      }],
+      ..default()
+    },
+  );
+
+  let output = CommandBuilder::new(format!(
+    "--chain regtest --index-runes wallet mint --fee-rate 1 --rune {} --postage 1000sat",
```
Comment (raphjaph) [inline tests/wallet/mint.rs:299]: maybe use 2222sat so it's visually different from the default

File: tests/wallet/mint.rs [hunk 4]
```diff
@@ -232,6 +234,8 @@ fn minting_rune_and_then_sending_works() {
     132_u128
   );
 
+  assert_eq!(balance.runic.unwrap(), 20000_u64);
```
Comment (raphjaph) [inline tests/wallet/mint.rs:237]:
```suggestion
  assert_eq!(balance.runic.unwrap(), 20000);
```

File: tests/wallet/mint.rs [hunk 5]
```diff
@@ -212,6 +212,8 @@ fn minting_rune_and_then_sending_works() {
     111_u128
   );
 
+  assert_eq!(balance.runic.unwrap(), 10000_u64);
```
Comment (raphjaph) [inline tests/wallet/mint.rs:215]:
```suggestion
  assert_eq!(balance.runic.unwrap(), 10000);
```

File: src/subcommand/wallet/mint.rs [hunk 6]
```diff
@@ -32,6 +37,8 @@ impl Mint {
       bail!("rune {rune} has not been etched");
     };
 
+    let postage = self.postage.unwrap_or(TARGET_POSTAGE);
```
Comment (raphjaph) [inline src/subcommand/wallet/mint.rs:40]: If the postage is beneath the dust limit it should return an errors. Something like this:

```
ensure!(
  destination.script_pubkey().dust_limit < postage,
  "postage below dust limit"
);
```

**Review by raphjaph — CHANGES_REQUESTED** [review]
Thanks for the PR! 

Left some comments

**Comment by ynohtna92** [comment]
Thanks for reviewing @raphjaph. I have made all those changes as requested and added a new test for the dust limit.

**Review by raphjaph — APPROVED** [review]
LGTM, great PR!

Just fixed some small things

**Pull request merged**